$MirOS: ports/security/gnupg/patches/patch-m4_tar-ustar_m4,v 1.1 2006/10/02 02:05:45 tg Exp $
--- m4/tar-ustar.m4.orig	Wed Sep 19 18:33:45 2012
+++ m4/tar-ustar.m4	Wed Jan  2 21:21:48 2013
@@ -18,6 +18,8 @@ AC_DEFUN([GNUPG_CHECK_USTAR],
      if test x$_do_tar = x ; then
         AC_PATH_PROG(TAR,"tar")
         _mytar=$ac_cv_path_TAR
+     else
+        _mytar=$_do_tar
      fi
 
      # Check if our tar is ustar format.  If so, it's good.  TODO: Add some
